pkg/nfs/utils.go

```diff
@@ -16,9 +17,10 @@ func NewDefaultIdentityServer(d *nfsDriver) *IdentityServer {
 	}
 }
 
-func NewControllerServer(d *nfsDriver) *ControllerServer {
+func NewControllerServer(d *nfsDriver, workingDir string) *ControllerServer {
 	return &ControllerServer{
-		Driver: d,
+		Driver:          d,
+		workingMountDir: workingDir,
```

Comment:
in current implementation, `workingMountDir` is always `/tmp`, so maybe we don't need this field, just use a const var `workingMountDir` is enough

Reply:
I believe it was parameterized this way to facilitate unit tests